selftests/unit/test_loader.py

```diff
@@ -327,7 +327,7 @@ def test_load_multiple_imports(self):
             'avocado_loader_unittest')
         avocado_multiple_imp_test.save()
         test_class, test_parameters = (
-            self.loader.discover(avocado_multiple_imp_test.path, True)[0])
+            self.loader.discover(avocado_multiple_imp_test.path, loader.ALL)[0])
```

Comment:
I'm wondering why we only check with `loader.ALL`. Never mind it seems to cover all occurrences, nice cleanup.